src/frozendict/__init__.pyi

```diff
@@ -0,0 +1,54 @@
+from collections.abc import Hashable
```

Comment:
Looks like `Hashable` is imported but not actually used?

Reply:
This is highly possible.